### ISSUE TITLE
Premium Node Garbage Collector

### ISSUE THREAD

**Issue by ItzOP** [Minehut/Meta#336]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**
In-game

**Describe the feature**
I've noticed that that all premium nodes on Minehut seem to run a worse garbage collector than the free nodes:
![image](https://user-images.githubusercontent.com/70059430/97115627-9a7fe580-16ef-11eb-977e-e3a7cef6de94.png)
As seen on that screenshot timings always comes up with the same message about the GC.
So I'm thinking it'd be nice to have an option to start servers with a different GC if possible, maybe give the ability to the user to change the GC before starting up the server. If not possible, then it'd be better if premium servers had the same GC as free servers. I don't know much about this stuff, the only thing I'm going off is the timings, therefore if I'm completely wrong about this feel free to close the request. 

**How the feature is useful**
An overall decrease in lag and better performance for premium nodes.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by drtshock** [comment]
They all run the same for the record.
